src/clarity-angular/data/tree-view/_tree-view.clarity.scss

```diff
@@ -14,232 +16,173 @@ $clr-tree-node-children-margin-big: $clr-tree-node-touch-target-big - $clr-tree-
 $clr-tree-node-touch-target-small: $clr_baselineRem_1;
 $clr-tree-node-children-margin-small: $clr-tree-node-touch-target-small - $clr-tree-caret-size / 2 - 1;
 
-//Checkbox in a TreeNode
-@mixin positionCheckBoxInTree(
-    $touch-target: $clr-tree-node-touch-target,
-    $checkbox-height: $clr-checkbox-radio-height,
-    $checkmark-left-position-offset: 4px) {
+$clr-tree-node-content-icon-margin-right: $clr-baselineRem_0_25;
+$clr-tree-node-link-padding-left: $clr-baselineRem_0_25;
 
-    & > clr-checkbox,
-    & > .checkbox {
-        $checkbox-label-center-position: ($touch-target - $checkbox-height) / 2;
+$clr-tree-node-caret-link-hover-color: $clr-black;
+$clr-tree-link-hover-color: $clr-custom-links-hover-color;
+$clr-tree-link-selection-color: $clr-selection-color;
+$clr-tree-link-text-color: clr-getTextColor();
 
-        //Positioning
-        left: $touch-target;
+$clr-tree-checkbox-padding-top: ($clr-tree-node-touch-target - $clr-checkbox-radio-label-height) / 2;
+$clr-tree-checkbox-padding-left: ($clr-tree-node-touch-target - $clr-checkbox-radio-height) / 2;
 
+@include exports('tree-view.clarity') {
+    .clr-tree-node {
         //Display
-        height: $touch-target;
-
+        display: block;
 
-        & > label {
-            height: $touch-target;
-            padding-left: $touch-target;
-        }
-
-        //I unfortunately had to do this to accommodate the click target sizing
-        //Checkbox container
-        input[type="checkbox"] + label::before {
-            top: $checkbox-label-center-position;
-            left: $checkbox-label-center-position;
-        }
-
-        //TODO: Apply the same changes in forms.clarity.scss
-        //Checkbox checkmark
-        input[type="checkbox"] + label::after {
-            top: $checkbox-label-center-position;
-            left: $checkbox-label-center-position + $checkmark-left-position-offset;
+        //Not sure why Firefox doesn't add this by default using the collapse
+        //animation directive.
+        @include fixForFirefox() {
+            overflow-y: hidden;
         }
     }
-}
-
-@mixin caret-touch-target($touch-target: $clr-tree-node-touch-target) {
-    height: $touch-target;
-    width: $touch-target;
-}
 
-@mixin spinner-positioning($touch-target: $clr-tree-node-touch-target) {
-    left: ($touch-target - $clr-tree-spinner-size) / 2;
-    top: ($touch-target - $clr-tree-spinner-size) / 2;
-}
-
-@mixin generate-tree-content($touch-target: $clr-tree-node-touch-target) {
+    .clr-tree-node-content-container {
+        //Display
+        display: flex;
+        align-items: center;
+    }
 
-    //Tree Node Content
     .clr-treenode-content {
-
+        //Note to people reviewing: Adding display flex to the content area. You think this might break a few things
+        //on the consumer side?
         //Display
-        line-height: $touch-target;
-        padding-left: $touch-target;
+        flex: 1 1 auto;
```

Comment:
This is the same as `flex: auto`. It should be ok as flexbox should size the content item(s) based on their height/width.

Reply:
I have a habit of writing the whole thing because of IE10. IE 10 default was flex: 1 0 auto instead of flex: 1 1 auto. I know we don't officially support IE10 but I think its good to specify explicitly.